syfertext/__init__.py

```diff
@@ -45,16 +52,25 @@ def load(pipeline_name: str) -> Language:
         # Get a copy of the pipeline using its pointer
         pipeline = result.get_copy()
 
+        # Save the pipeline to local storage
+        save(pipeline_name=pipeline_name, pipeline=pipeline, destination="local")
+
     elif isinstance(result, Pipeline):
 
         # In this case, the Pipeline object is found on the local worker
         # which is a virtual worker by default as of the current PySyft version
         # 0.2.9. We do not consider that it is officially deployed.
         deployed_on = None
 
+        print("Pipeline found")
```

Comment:
let's remove this message for now, in future we will add better logging for complete syfertext